### PR TITLE
Created Avatar Component

### PR DESCRIPTION
**Pull request by khyatibaria**
###  Please Review my Avatar Component
image

### CODE REVIEW

**Review by pavan6115 — APPROVED** [review]
Looks good 👍

File: avatar/avatar.css [hunk 1]
```diff
@@ -0,0 +1,71 @@
+:root {
+  --primary-bgcolor: #8b6aee;
+  --padding-md: 1rem;
+  --margin-md: 1rem;
+  --fontSize: 1rem;
+  --radius: 0.25rem;
+}
```
Comment (yashpatel1797) [inline avatar/avatar.css]: you can declare this in a common CSS file and use it everywhere.

File: avatar/avatar.html [hunk 2]
```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (yashpatel1797) [inline avatar/avatar.html:5]: here you can add author meta tag

**Review by yashpatel1797 — APPROVED** [review]
code looks good to me. added some suggestions

**Pull request merged**